### PR TITLE
Update validator-node.md

### PR DESCRIPTION
**Pull request by YazzyYaz**

### CODE REVIEW

File: docs/nodes/validator-node.md [hunk 1]
```diff
@@ -279,8 +279,8 @@ You have successfully set up a bridge node that is syncing with the network.
 
 ### Setup QGB keys
 
-This step helps get you prepared for when the Quantum Gravity Bridge
-is ready to be deployed. You would still need to go through this step
+This step helps get you prepared for when the Quantum Gravity Bridge relayer
+is deployed by Celestia Labs. You would still need to go through this step
```
Comment (SweeXordious) [inline docs/nodes/validator-node.md:283]: After thinking about it, we can either leave it as is or replace with the following: 

```suggestion
This step helps get you prepared for when the Quantum Gravity Bridge
is enabled. You would still need to go through this step
```

File: docs/nodes/validator-node.md [hunk 2]
```diff
@@ -358,3 +358,9 @@ txhash: <tx-hash>
 ```
 
 You should now be able to see your validator from a block explorer like [here](https://celestia.explorers.guru)
+
+## Run the QGB Orchestrator
+
+Now that the QGB will be enabled for BSR,  we should run the QGB
+orchestrator to be able to sign attestations. To run it, please
+check the docs [here](https://docs.celestia.org/nodes/qgb-orchestrator/#install-the-qgb-binary).
```
Comment (SweeXordious) [inline docs/nodes/validator-node.md:366]:
```suggestion
check the docs [here](https://docs.celestia.org/nodes/qgb-orchestrator/#how-to-run).
```

File: docs/nodes/validator-node.md [hunk 3]
```diff
@@ -358,3 +358,9 @@ txhash: <tx-hash>
 ```
 
 You should now be able to see your validator from a block explorer like [here](https://celestia.explorers.guru)
+
+## Run the QGB Orchestrator
+
+Now that the QGB will be enabled for BSR,  we should run the QGB
```
Comment (SweeXordious) [inline docs/nodes/validator-node.md:364]:
```suggestion
Now that the QGB will be enabled for Blockspace Race (BSR),  all validators will need to run the QGB
```

**Review by SweeXordious — APPROVED** [review]
Thanks a lot for handling this.

Once we have the bootstrappers, maybe we should add them also here or link to them somewhere

**Review by jcstein — APPROVED** [review]

**Pull request merged**